### ISSUE TITLE
LengthIsEvenNumberOfHexDigits not implemented when using new lines in hex literals

### ISSUE THREAD

**Issue by ScanMountGoat** [inspier/hexlit#2]
The ability to write hex literals without quotes and retain some syntax highlighting is a really neat feature. I noticed something strange happening when using multiple lines. The check for even or odd length seems to change depending on how many new lines are present even if the number of hex literals remains the same. This is on version 0.5.2 from crates.io.

```rust
#[test]
fn test_new_lines() {
    use hexlit::hex;

    // Doesn't compile.
    let a = hex!(
        00000000 00000000 00000000 00000000
        00000000 00000000 00000000 00000000 00000000
    );

    // These compile.
    let b = hex!(
        00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
    );

    let c = hex!(00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000);
}
```

**Comment by inspier** [comment]
Much thanks for reporting this, always happy to support additional input formats. This issue should be fixed by [9c62391](https://github.com/inspier/hexlit/commit/9c623912d1b9b864cf3b376d07a5037f31efb726) in [0.5.3](https://github.com/inspier/hexlit/releases/tag/0.5.3). Let me know if it still isn't working for you.

**Comment by ScanMountGoat** [comment]
I just finished porting my tests over. I can confirm it's working now on 0.5.3. Thank you for looking into this.
